userguide/system.rst

```diff
@@ -974,19 +974,25 @@ controller users and groups.
    System Dataset Screen
 #endif truenas
 
-.. note:: Encrypted, locked volumes are not displayed in the
-   :guilabel:`System dataset pool` drop-down menu.
 
 The system dataset can optionally be configured to also store the
 system log and :ref:`Reporting` information. If there are lots of log
 entries or reporting information, moving these to the system dataset
 will prevent :file:`/var/` on the device holding the operating system
 from filling up as :file:`/var/` has limited space.
 
-Use the drop-down menu to select the ZFS volume (pool) to contain the
-system dataset. Whenever the location of the system dataset is
-changed, a pop-up warning indicates that the SMB service must be
-restarted, causing a temporary outage of any active SMB connections.
+Use the :guilabel:`System dataset pool` drop-down menu to select the ZFS
```

Comment:
```suggestion
Use the :guilabel:`System dataset pool` drop-down menu to select the
```

Reply:
The system dataset can be moved to unencrypted volumes (pools) or encrypted volumes which do not have passphrases. If the system dataset is moved to an encrypted volume, that volume is no longer allowed to be locked or have a passphrase set.